androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/Utils.java

```diff
@@ -99,4 +102,41 @@ public static Bitmap convertDrawableToBitmap(Context context, int drawableId) {
         drawable.draw(canvas);
         return bitmap;
     }
-}
\ No newline at end of file
+
+    /**
+     * Verifies if the application has the required meta-tag, `web_manifest_url`, and the
+     * `web_app_manifest.json` file, required to enable the Trusted Web Activity on Chrome OS.
+     * Important: The content the meta-tag and the JSON fire are not validated.
+     */
+    public static boolean hasChromeOsSupport(Context context) {
+        // We first check if /app/res/web_app_manifest.json is available
```

Comment:
nit: You're missing full stops on comments (in a few other places as well).

Reply:
Added.